packages/create-next-app/helpers/get-pkg-manager.ts

```diff
@@ -1,25 +1,17 @@
-import { execSync } from 'child_process'
-
 export type PackageManager = 'npm' | 'pnpm' | 'yarn'
```

Comment:
Nitpick, but this could actually just be:

```ts
export type PackageManager = 'npm' | 'pnpm' | 'yarn'

export function getPkgManager(): PackageManager {
  const userAgent = process.env.npm_config_user_agent

  if (userAgent?.startsWith('yarn')) {
    return 'yarn'
  } else if (userAgent?.startsWith('pnpm')) {
    return 'pnpm'
  }
  return 'npm'
}
```